Libs/MRML/Core/vtkMRMLAbstractViewNode.h

```diff
@@ -298,6 +299,18 @@ class VTK_MRML_EXPORT vtkMRMLAbstractViewNode : public vtkMRMLNode
   vtkSetMacro(ScreenScaleFactor, double);
   //@}
 
+  //@{
+  /// \brief Get/Set the displayable manager group associated with the view.
```

Comment:
For both Set/Get, could you add a note indicating that there are no consistency checks done. 

The developer is responsible to make sure the group associated with the view node is effectively associated with RendererWindow/Renderer set on the view.